scripts/loadWebhooks.js

```diff
@@ -0,0 +1,35 @@
+global.fetch = require('node-fetch').default
+const fs = require('fs')
+const { default: CozyClient, Q } = require('cozy-client')
+
+const newWebhooks = require('../assets/webhooks.json')
+
+const NODES_DOCTYPE = 'io.cozy.dissec.nodes'
+
+const main = async () => {
+  // Connect to the instance
+  const client = new CozyClient({
+    uri: process.argv[2],
+    schema: {
+      nodes: {
+        doctype: NODES_DOCTYPE,
+        attributes: {},
+        relationships: {}
+      }
+    },
+    token: process.argv[3]
+  })
+
+  // Fetch old nodes
+  const { data: oldNodes } = await client.collection(NODES_DOCTYPE).all()
```

Comment:
You should use `.collection` only when it's not possible directly through CozyClient methods. In this case, you can simply do: `await client.queryAll(Q(NODES_DOCTYPE))` : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L944